seed/seed.json

```diff
@@ -134,6 +134,122 @@
                 "channel": ["NIGHTLY", "BETA"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "AdNotificationsDisabledOnBetaAndReleaseChannelsStudy",
+            "experiments": [
+                {
+                    "name": "AdNotifications",
+                    "probability_weight": 100,
+                    "parameters": [
+                        {
+                            "name": "should_show_custom_notifications",
+                            "value": "false"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["AdNotifications"],
+                        "disable_feature": []
```

Comment:
`AdNotificationsDisabledOnBetaAndReleaseChannelsStudy`
should the feature be disabled? the name and the actual result doesn't match.

Reply:
@goodov same as below, this is due to the unusual case of having introduced the feature only now. The feature should be enabled for all groups, the group split is along the parameter `should_show_custom_notifications`.

Reply:
I will rename now. Thanks

Reply:
Fixed